portal-web/test/functional/com/liferay/portalweb/macros/FormPortlet.macro

```diff
@@ -59,6 +59,14 @@ definition {
 		FormPortlet.clickSubmit();
 	}
 
+	macro submitAndViewSuccessPage {
```

Comment:
@Jamersonze, can you please rebase this pull and use the recent added macro `submitSuccessfullyAndViewSuccessPage`?

Thanks.

Reply:
The macro "submitAndViewSuccessPage" is merged as well. Thank you.